### PR TITLE
*: cut v2.6.1

### PR DESCRIPTION
**Pull request by simonpasquier**
cc @juliusv @brian-brazil @gouthamve @SuperQ

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,19 @@
+## 2.6.1 / 2019-01-15
+
+* [BUGFIX] Azure SD: Fix discovery getting stuck sometimes. #5088
+* [BUGFIX] Marathon SD: Use `Tasks.Ports` when `RequirePorts` is `false`. #5026
+* [BUGFIX] Promtool: Fix "out-of-order sample" errors when testing rules. #5069
+
 ## 2.6.0 / 2018-12-17
 
 * [CHANGE] Promtool: Remove the `update` command. #3839
+* [CHANGE] Remove default flags from the container's entrypoint, run Prometheus from `/etc/prometheus` and symlink the storage directory to `/etc/prometheus/data`. #4976
```
Comment (brian-brazil) [inline CHANGELOG.md]: This should probably go first

**Comment by brancz** [comment]
:+1:

File: CHANGELOG.md (hunk 1 above)
Comment (brancz) [reply CHANGELOG.md]: agreed

**Comment by juliusv** [comment]
:+1:

**Comment by beorn7** [comment]
You still need to change `VERSION`

**Pull request merged**